janusgraph-test/src/main/java/org/janusgraph/graphdb/JanusGraphBaseTest.java

```diff
@@ -66,12 +68,17 @@
     public StoreFeatures features;
     public JanusGraphTransaction tx;
     public JanusGraphManagement mgmt;
+    private static final JanusGraphManager graphManager;
 
     public Map<String,LogManager> logManagers;
 
     public JanusGraphBaseTest() {
     }
 
+    static {
+        graphManager = new JanusGraphManager(new Settings());
```

Comment:
Having this initialization in multiple test classes might be a problem if running tests in parallel or reusing forks. Can you support default initialization for tests in `JanusGraphManager.getInstance()` (maybe adding a `getInstance(boolean forceCreate)` signature)?